commands/backport_records.py

```diff
@@ -78,7 +78,13 @@ def backport_records(event, context, **kwargs):
     # Delete the records missing from source.
     to_delete = dest_records_by_id.values()
 
-    if (len(to_create) + len(to_update) + len(to_delete)) == 0:
+    is_behind = (len(to_create) + len(to_update) + len(to_delete)) > 0
+    has_pending_changes = is_behind
+    if not is_behind:
+        data = dest_client.get_collection()["data"]
+        has_pending_changes = data.get("status") != "signed"
```

Comment:
Looking at the [`normandy-recipes` collection](https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/normandy-recipes), I don't see a `status` field in the API response. Where does this come from?

Reply:
Very good point, and that was partly a source of confusion when I investigated the issue.

The destination collection is `main-workspace/normandy-recipes` and not `main/normandy-recipes` (I was running the lambda with the wrong destination collection). The status attribute is set on the workspace collection, along with other metadata like reviewer comment etc.  which are not copied over the main one. Only [some attributes](https://github.com/Kinto/kinto-signer/blob/master/kinto_signer/updater.py#L374-L377) are copied.

I'll add a comment.